contracts/pool/src/methods/fix_limit/withdraw.rs

```diff
@@ -41,7 +41,7 @@ pub fn withdraw(
         .mul_int(collat_balance)
         .ok_or(Error::MathOverflowError)?;
 
-    let (underlying_to_withdraw, s_token_to_burn) = if amount == i128::MAX {
+    let (underlying_to_withdraw, s_token_to_burn) = if amount >= underlying_balance {
```

Comment:
Do we need the same change in methods/withdraw.rs ?

Reply:
Nice catch! Thanks!